### PR TITLE
RBF UI adjustments

### LINKED ISSUE #2911: UI2: wallet backup fix padding
Current implementation: 
current

Fixed screen: 
image

This fix should be done in every flow where users see seed words on the screen.

### LINKED ISSUE #2991: RBF screens copy adjustment
Edit: 
image

### PR DESCRIPTION
**Pull request by mmilata**
Fixes #2911, fixes #2991.

* [master diff](https://satoshilabs.gitlab.io/-/trezor/trezor-firmware/-/jobs/4345498859/artifacts/master_diff/index.html)
* [differing screens](https://satoshilabs.gitlab.io/-/trezor/trezor-firmware/-/jobs/4345498859/artifacts/master_diff/differing_screens.html)

7f1709c206c14cfdf408f2cf6aa1aeecfb1f740517112785336f7120c0d96fb8 c021e2acf2d57c1743d2e7e7a3e77695678d39ee6fd3d07949273e87351159a9 efa8847539ff616225667c8a9cc64ffa079e8a8ac72c1f36933793255e8ef651 c7df9f9db95635f6d118fc6add21593992fc954de336fb6bdb20d32d696c5561 aab3ff25d61f07262a519887d696c0fa4bc3b2c10600046b7e836fcc7b42b352 11fd88ab7c1c861939515b73cfc58bc70972b0d5babad341b6dfe8ca2067e74a 676cea55e346dac72b78a3fe1702818515950783019e99c27777528350cd668c f60e53aa7ab64bc907a8a98e8f60bcf8ff398ccf9fffd2cd38ece8cea7490e35  77cb86e1e59cdb32bcf59d8c3af3b8671cd2bf63c3a45930a70a9f8a54b92f22

### CODE REVIEW

**Comment by Hannsek** [comment]
Shouldn't we add tests for the `i` screen?

File: core/src/trezor/ui/layouts/tt_v2/__init__.py [hunk 1]
```diff
@@ -996,45 +1001,93 @@ async def confirm_modify_output(
...
+        "modify_fee",
+        ButtonRequestType.SignTx,
+    )
+
+    while True:
+        result = await ctx.wait(layout)
+
+        if result is CONFIRMED:
+            return
+        elif result is INFO:
```
Comment (grdddj) [inline core/src/trezor/ui/layouts/tt_v2/__init__.py:1075]: There seem to be no tests for this `INFO` button

File: core/src/trezor/ui/layouts/tt_v2/reset.py [hunk 2]
```diff
@@ -20,15 +20,19 @@
...
     current = ""
+    fill = 2
 
     for i, word in enumerate(share_words):
         if i % per_page == 0:
             if i != 0:
                 pages.append(current)
             current = ""
+
+            lastnum = i + per_page + 1
```
Comment (grdddj) [inline core/src/trezor/ui/layouts/tt_v2/reset.py:31]: I would welcome a comment on why we do this

File: core/embed/rust/src/ui/model_tt/layout.rs [hunk 3]
```diff
@@ -1652,8 +1655,10 @@ pub static mp_module_trezorui2: Module = obj_module! {
 
     /// def show_spending_details(
     ///     *,
-    ///     account: str,
+    ///     title: str | None,
```
Comment (grdddj) [inline core/embed/rust/src/ui/model_tt/layout.rs:1658]: It looks like the type here should be rather `title: str = "PIN INFORMATION"`, as the title is in the end always a string, even when you input `None`.

That would require changing the code to 
```rs
let title: StrBuffer = kwargs.get_or(Qstr::MP_QSTR_title, "INFORMATION".into())?
```

The same with `fee_rate_title` below - `: str = "Fee rate:"`

**Review by grdddj — COMMENTED** [review]
Looks alright, I have just some small requests

File: core/src/trezor/ui/layouts/tt_v2/__init__.py (hunk 1 above)
Comment (mmilata) [reply core/src/trezor/ui/layouts/tt_v2/__init__.py]: Fixed in [c0cfca8](https://github.com/trezor/trezor-firmware/pull/3029/commits/c0cfca8a39ed8bb5da2959741cb3dbdf11deab29).

File: core/src/trezor/ui/layouts/tt_v2/reset.py (hunk 2 above)
Comment (mmilata) [reply core/src/trezor/ui/layouts/tt_v2/reset.py:32]: Fixed in [c0cfca8](https://github.com/trezor/trezor-firmware/pull/3029/commits/c0cfca8a39ed8bb5da2959741cb3dbdf11deab29).

File: core/embed/rust/src/ui/model_tt/layout.rs (hunk 3 above)
Comment (mmilata) [reply core/embed/rust/src/ui/model_tt/layout.rs]: Fixed in [c0cfca8](https://github.com/trezor/trezor-firmware/pull/3029/commits/c0cfca8a39ed8bb5da2959741cb3dbdf11deab29).

**Comment by mmilata** [comment]
Info screen test: https://satoshilabs.gitlab.io/-/trezor/trezor-firmware/-/jobs/4368797321/artifacts/master_diff/added/TT-device_tests-bitcoin-test_signtx.py::test_information_replacement.html

**Review by grdddj — APPROVED** [review]
LGTM

**Comment by bosomt** [comment]
**QA OK**

**Info:**
 - Suite version: desktop 23.6.1 (bddd516378beadfcff94523e7c94834ba14f147f)
 - Browser: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) TrezorSuite/23.6.1 Chrome/112.0.5615.183 Electron/24.3.1 Safari/537.36
 - OS: MacIntel
 - Screen: 1440x900
 - Device: model T 2.6.1 Universal (revision 089ff768229f543c89927ff34c4d45d830a2442f)
 - Transport: BridgeTransport 2.0.33

**Pull request merged**